### PR TITLE
UI Adjustments

### PR DESCRIPTION
**Pull request by kkatzen**
1. Moving charts into Cards/ folder 
2. Adding additional styling to breadcrumbs, a few other places (swapped disparity chart colors)
3. Fixed useResponsiveWidth type checking issues
4. Added missing data warning for disparity chart (via age/gender charts)
5. Adding responsive width to disparity chart
6. Opening data source info in a new tab

### CODE REVIEW

File: app/src/features/charts/DisparityBarChart.tsx [hunk 1]
```diff
@@ -211,19 +215,23 @@ function TwoVarBarChart(props: {
...
-        props.data,
-        props.breakdownVar,
-        props.breakdownVarDisplayName,
-        props.thickMeasure,
-        props.thickMeasureDisplayName,
-        props.thinMeasure,
-        props.thinMeasureDisplayName
-      )}
-    />
+    <div ref={ref as RefObject<HTMLDivElement>}>
```
Comment (aaronsn) [inline app/src/features/charts/DisparityBarChart.tsx:220]: Is it possible to bake the return type into the useResponsiveWidth so it doesn't require casting?

File: app/src/features/charts/DisparityBarChart.tsx [hunk 2]
```diff
@@ -211,19 +215,23 @@ function TwoVarBarChart(props: {
...
+      <Vega
+        spec={getSpec(
+          props.data,
+          width ? width : 100, // Set a default value until width is set
```
Comment (aaronsn) [inline app/src/features/charts/DisparityBarChart.tsx:224]: Perhaps useResponsiveWidth should accept a default value on initialization

File: app/src/features/charts/UsaChloroplethMap.tsx [hunk 3]
```diff
@@ -184,13 +182,17 @@ function UsaChloroplethMap(props: {
...
+      ref={ref as RefObject<HTMLDivElement>}
       style={{
         width: "80%",
         margin: "auto",
       }}
     >
-      <Vega spec={spec} width={width} signalListeners={props.signalListeners} />
+      <Vega
+        spec={spec}
+        width={width ? width : 100} // Set a default value until width is set
```
Comment (aaronsn) [inline app/src/features/charts/UsaChloroplethMap.tsx:193]: Same as above about default value

File: app/src/features/cards/DisparityBarChartCard.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+              breakdownVarDisplayName={props.breakdownVarDisplayName}
+            />
+            <LinkWithStickyParams
+              target="_blank"
+              to={`${DATA_CATALOG_PAGE_LINK}?${DATASET_PRE_FILTERS}=${props.datasetIds.join(
+                ","
+              )}`}
+            >
+              View Data Sources
+            </LinkWithStickyParams>
```
Comment (aaronsn) [inline app/src/features/cards/DisparityBarChartCard.tsx:56]: At some point, can we pull this out and make it a common element for all reports per the mocks?

**Review by aaronsn — APPROVED** [review]

File: app/src/features/charts/DisparityBarChart.tsx (hunk 1 above)
Comment (kkatzen) [reply app/src/features/charts/DisparityBarChart.tsx:220]: oh ha I actually had already done that I guess I forgot to remove this cast

File: app/src/features/charts/DisparityBarChart.tsx (hunk 2 above)
Comment (kkatzen) [reply app/src/features/charts/DisparityBarChart.tsx:224]: ya good call!

File: app/src/features/cards/DisparityBarChartCard.tsx (hunk 4 above)
Comment (kkatzen) [reply app/src/features/cards/DisparityBarChartCard.tsx:56]: ya! i'll put in a CardWrapper class in a followup

**Pull request merged**